library/options.inc.php

```diff
@@ -184,7 +184,7 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
 			
 				$optionValue = attr($lrow ['option_id']);
 			
-				if ($multiple && (strlen ( $currvalue ) == 0 && $lrow_backup ['is_default']) || 
```

Comment:
Also see a bug near here. Gonna fix this and include it in your commit (since you inadvertently identified this) before bringing into the codebase.